### PR TITLE
DR-1623 Use correct SAM endpoint for each environment

### PR DESCRIPTION
**Pull request by tlangs**
We need to not use SAM dev for each environment. Once https://github.com/DataBiosphere/jade-data-repo/pull/891 is merged, the API will provide `samUrl` in its response from the `configuration` endpoint, which we can then use to talk to the correct SAM deployment.

### CODE REVIEW

File: src/sagas/repository.js [hunk 1]
```diff
@@ -441,8 +442,9 @@ export function* countResults({ payload }) {
 
 export function* getFeatures() {
   try {
-    // TODO: look up from the correct Sam environment
-    const url = 'https://sam.dsde-dev.broadinstitute.org/api/groups/v1';
+    const samUrl = yield select(getSamUrl);
+    const url = `${samUrl}/api/groups/v1`;
+    console.log(url);
```
Comment (snf2ye) [inline src/sagas/repository.js]: Remove before merging?

**Review by snf2ye — APPROVED** [review]
One suggestion, but LGTM - nice job!!

File: src/sagas/repository.js (hunk 1 above)
Comment (tlangs) [reply src/sagas/repository.js]: doh!

**Review by pshapiro4broad — APPROVED** [review]

**Review by fboulnois — APPROVED** [review]
straightforward, lgtm

**Pull request merged**